src/object.d

```diff
@@ -482,6 +482,167 @@ unittest
     assert(a1 != a2);
 }
 
+/**
+Destruct the given object. Correctly call its destructor or finalizer so it no longer
+references any other objects. This $(I does not) reset the memory to init state. After
+this call succeeds, the object memory is in an invalid state, and should not be referenced.
+It does $(I not) initiate a GC cycle or free any GC memory.
+*/
+void destruct(T)(ref T obj) if (is(T == struct))
+{
+    _destructRecurse(obj);
+}
+
+///
```

Comment:
If you write `ditto` here Ddoc will collect this function with the one above to be an overload, i.e. it will group them together.

Reply:
Oh okay. Weird rule.